packages/xstate-react/src/createActorContext.tsx

```diff
@@ -1,38 +1,80 @@
 import * as React from 'react';
-import { RestParams, useInterpret } from './useInterpret';
+import { useInterpret } from './useInterpret';
 import { useActor as useActorUnbound } from './useActor';
 import { useSelector as useSelectorUnbound } from './useSelector';
-import { ActorRefFrom, AnyStateMachine, EmittedFrom } from 'xstate';
+import {
+  ActorRefFrom,
+  AnyStateMachine,
+  AreAllImplementationsAssumedToBeProvided,
+  EmittedFrom,
+  InternalMachineOptions,
+  InterpreterOptions,
+  Observer,
+  StateFrom
+} from 'xstate';
+import { UseMachineOptions } from './useMachine';
 
 export function createActorContext<TMachine extends AnyStateMachine>(
   machine: TMachine,
-  ...[options = {}, observerOrListener]: RestParams<TMachine>
+  interpreterOptions?: InterpreterOptions,
+  observerOrListener?:
+    | Observer<StateFrom<TMachine>>
+    | ((value: StateFrom<TMachine>) => void)
```

Comment:
this is a little bit of a breaking change for this test case:
https://github.com/statelyai/xstate/blob/6f15b2c0cf33c70f97e91ecaba5a250f0b68011a/packages/xstate-react/test/createActorContext.test.tsx#L307-L338

Reply:
just rechecking - are we OK with this breaking change?

Reply:
Yes; options was in the wrong place